### PR TITLE
Add firewall FAQ for TTIG 

### PR DESCRIPTION
**Pull request by JackGruber**
Information to run the TTIG behind a firewall

### CODE REVIEW

**Comment by jpmeijers** [comment]
```
| IPv4       | UDP      | See [Router addresses](/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses) for your region | 1700 | Semtech UDP Packet Forwarder |
```

Is that port really necessary? As far as I understood this gateway speaks websockets over TCP 443, and not UDP port 1700 like older semtech protocol gateways.

**Comment by JackGruber** [comment]
I just did another network trace, and you're right. I probably mixed up the rules with my second gateway. I have removed the Semtech UDP port.

File: _content/gateways/thethingsindoor/faq.md [hunk 1]
```diff
@@ -60,3 +60,15 @@ This is definitely not recommended since you cannot recreate a gateway with the
...
 If the TTIG is reset, only the WiFi and the LNS credentials will be deleted. It will retain its EUI and CUPS creds and will contact the CUPS server to obtain the LNS creds once the WiFi if configured.
 So reseting it (with the reset button) *will not affect* your registration of the gateway on the network.
+
+#### Q. I want to operate the gateway behind a firewall
+
+The following connections must be permitted in the firewall, the LNS address depends on your selected router in the Gateway configuration. 
+
+| IP Version | Protocol | Destination                                            | Port | Description |
+| ---------- | -------- | :----------------------------------------------------- | ---- | ----------- |
+| IPv4       | TCP      | lns.eu.thethings.network<br />lns.us.thethings.network | 443  | LNS         |
```
Comment (KrishnaIyer) [inline _content/gateways/thethingsindoor/faq.md]: Please use `lns.{eu|us|au|br|as}.thethings.network` to make this brief and future proof..  :) 
>  the LNS address depends on your selected router in the Gateway configuration. 

This is technically not true. All EU868 gateways currently only connect to EU and so on.

Comment (JackGruber) [reply _content/gateways/thethingsindoor/faq.md]: OK, i have pushed the changes

**Comment by KrishnaIyer** [comment]
Thanks @JackGruber

**Pull request merged**